charts/local-kafka/templates/kafka.yaml

```diff
@@ -17,6 +17,8 @@ spec:
       failureThreshold: 30  # 30 * 10s
     authorization:
       type: simple
+      superUsers:
+        - ANONYMOUS
```

Comment:
ran into an issue (`User:ANONYMOUS is Denied Operation`) where without tls, the user is `ANONYMOUS`, which isn't allowed to do anything currently. not sure if there's a better way to give it permissions though?

Reply:
I think this is fine, it's local.